src/containers/ContractPage/ContractPage.js

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled from '@emotion/styled';
+import './ContractPage.css';
+import SearchBar from '../../components/SearchBar';
+import TeamSection from '../../components/TeamSection';
+import SideBar from '../../components/SideBar';
+import { getOne, saveNegotiation } from '../../redux/actions';
+import NavBar from '../../components/NavBar';
+import { negotiationSchema } from '../../redux/middlewares/schemas/schemas';
+import EditorView from '../../components/EditorView/EditorView';
+import woofmark from 'woofmark';
+import { gabeImg } from './gabe-image.jpg';
+
+// eslint-disable-next-line
+class ContractPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.toggleView = this.toggleView.bind(this);
```

Comment:
Related to @babel/plugin-proposal-class-properties, you can define `toggleView` as a class property and creating it as an arrow function so it creates no context.